Fixed #38: Corrected wrong date format

hypfvieh opened the pull request.

oknozor commented:
Could you add [suggested clippy fix](https://github.com/oknozor/musicbrainz_rs/pull/39/checks?check_run_id=2389074464) ?

hypfvieh commented:
I don't think that my changes messed up clippy. I just removed the % sign using vi. 
The clippy error was already present in your previous [commit](https://github.com/oknozor/musicbrainz_rs/runs/2321735569).
I fixed the suggestions. It reported "warnings" because of semicolons on lines I didn't change (e.g. in browser_deserializer.rs)

oknozor commented:
Sorry I didn't notice before but can you rewrite your commit messages to respect the [conventional commit]((https://www.conventionalcommits.org/en/v1.0.0-beta.2/#summary) specification ?
